### ISSUE TITLE
Help collecting accessibility support data

### ISSUE THREAD

**Issue by Elchi3** [openwebdocs/project#193]
### Problem statement

The non-profit OWD collective currently contributes heavily to mdn/browser-compat-data. The BCD project collects compatibility data for browsers and some runtimes like nodejs and deno. However, the BCD project doesn't collect support data for accessibility necessarily (see https://github.com/mdn/browser-compat-data#what-isnt-tracked)

Several issues were brought up about accessibility support recently, though (non-exhaustive list)
- https://github.com/mdn/browser-compat-data/issues/21847
- https://github.com/mdn/browser-compat-data/issues/21648
- https://github.com/mdn/browser-compat-data/issues/21663

The [web-features](https://github.com/web-platform-dx/web-features) project ("baseline"), doesn't yet take accessibility support info into account due to missing data. See https://github.com/web-platform-dx/web-features/issues/498 and https://github.com/web-platform-dx/web-features/pull/519.

Web developers are not informed properly about a11y support in browsers and in assistive technologies.

### Proposed solutions

There are probably a few things we could do. Some initial thoughts:

1) We sometimes use `partial_implementation` and `note` in BCD to say that some aspects of a feature don't work. See ["`<hr> `in select"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr#browser_compatibility) as an example for this. However, there are no BCD data guidelines about these partial support statements. We could propose to create a BCD data guideline, so that BCD can talk about at least some of the a11y support issues in a consistent way.

2) We could start contributing to https://github.com/accessibilitysupported/a11ysupport.io (licensed CC BY 4.0)
From https://a11ysupport.io/faq:
> This is a community-driven website that aims to help inform developers about what code features (roles, states, properties, elements, etc.) are supported by assistive technologies, and what that support looks like.
It might take a bit to get familiar with this data and with testing AT. https://a11ysupport.io/contribute has some pointers. 

3) We could look into https://github.com/web-platform-tests/interop-accessibility/ which is about improving the interop for accessibility and maybe that group has more research that ought to be surfaced in data and in documentation.

4) Finally, we should probably not only figure out how to curate and record good accessibility support but also figure out the data quality or other requirements needed to be met so that MDN/Caniuse/Baseline/others would consider the data good enough to be displayed and shown to web developers (prominently).

### Task list

As written, the problem statement and the work on the proposed solutions is really, really large and high effort. There is a lot to unpack here. I think the OWD collective currently won't be able to do all of this. Maybe some allies in the a11y community could help. Let me try to make up a task list (this will probably change if we get into this work):

- [ ]  Research how we currently record a11y support in BCD (`<hr>` in select is just one example) and use that research to propose a data guideline for BCD to talk about a11y support consistently.
- [ ] Figure out if we could contribute to https://github.com/accessibilitysupported/a11ysupport.io. Probably manually for now, maybe later we could investigate if our compat data automation project (https://github.com/openwebdocs/mdn-bcd-collector) could be utilized as well, or if there are learnings from it that a11ysupport.io could use.
- [ ] Talk to https://github.com/web-platform-tests/interop-accessibility/ to see if the group has data that we should record as a11y support data
- [ ] Look for help/contributors in the a11y community to help with this support data collection.
- [ ] Once more familiar, present all our research to the WebDX group to reconsider inclusion of a11y support data in baseline.

### Priority assessment

- **Effort**: XL (probably several months of work)
- **Dependencies**: https://github.com/accessibilitysupported/a11ysupport.io is pretty new to me and I think to anyone from OWD. So, maybe this project depends a bit on how well we can contribute to a11ysupprt.
- **Community enablement**: Hopefully there are a11y community who would help or at least advise us with this project, so I would say yes, we will hopefully be able to enable some folks to join us on this work.
- **Momentum**: The issue on the WebDX repo had 30 upvotes. The a11y interop project seems pretty new, too. 
- **Enabling learners**: n/a
- **Enabling professionals**: n/a 
- **Underrepresented topics / Ethical web**: Yes, pretty much!
- **Operational necessities**: n/a
- **Addressing needs of the web industry**: I think so. I have no other data/surveys that backs up the (high) need of this project, but we could try to get some more info from a11ysupport.io or we could run a survey on our own.

### More information

Open Web Docs (OWD) is a non-profit collective funded by corporate and individual donations.

In order for this project to happen, please consider donating to OWD on https://opencollective.com/open-web-docs.
For more information on sponsorship and membership tiers, see https://openwebdocs.org/membership/

More information is available at https://openwebdocs.org/.
For questions, please reach out to [florian@openwebdocs.org](mailto:florian@openwebdocs.org).

**Comment by Elchi3** [comment]
Via @dontcallmedom: https://aria-at.w3.org/reports

**Comment by Elchi3** [comment]
Via Steve Faulkner: https://freedomscientific.github.io/standards-support/html.html

**Comment by Elchi3** [comment]
This is now lead by Lola, see https://lolaslab.co/blog/2025/acd-2025-report/ for the latest report. I'm leaving this open as OWD still wants to help Lola's ACD initiative.
